src/plugins/Plugin.js

```diff
@@ -87,9 +87,9 @@ module.exports = class Plugin {
       this.el = plugin.render(this.core.state)
       targetElement.appendChild(this.el)
 
-      this.target = targetElement
+      this.target = this.el
```

Comment:
I'm not sure what the effects of this change would be. if we do this we could remove `this.target` entirely and use `this.el` everywhere instead.

It seems like these two lines could stay the same. Or is there a reason to change this here that I'm missing?

Reply:
Without this change, `this.target` in `Webcam` mounted into `Dashboard`, for example, becomes `<body>`. With this change, `this.target` becomes `<div class="UppyDashboard">`. 

Yes, we could just use `this.el` instead of `this.target` in Webcam, Dashboard and other place if we need. I thought maybe `target` is more consistent with what we have in options (`.use(Webcam, { target: '.Webcam' })`) and all that.

Reply:
oh right, that was it, thanks for clarifying. yeah i think ideally we would have a single property for this, whether it's called `this.el` or `this.target`. `this.target` is good for symmetry with options as you say and would also require fewer changes so i'm cool with that.

Reply:
On the other hand, `this.target` should be a container, so if Dashboard’s target is `<body>`, this is it. And `this.el` is the actual root element of the plugin, so `<div class="UppyDashboard">` 🤔

Reply:
should we only store `this.el` then and remove `this.target` entirely? the latter is only used in `mount()`, so a simple variable would suffice, and when we've done `this.target.querySelector(xyz)` in plugins before, we actually really meant `this.el.querySelector(xyz)` anyway.